package.json

```diff
@@ -12,13 +12,14 @@
   },
   "homepage": "https://thelounge.chat/",
   "scripts": {
-    "coverage": "nyc mocha",
+    "coverage": "rm -rf .nyc_output/ && npm-run-all test:* && nyc --nycrc-path=test/.nycrc-report report",
     "start": "node index start",
     "start-dev": "npm-run-all --parallel watch start",
     "build": "webpack",
     "watch": "webpack --watch",
     "test": "npm-run-all --aggregate-output --parallel --continue-on-error test:* lint:*",
-    "test:mocha": "mocha --colors",
+    "test:mocha": "nyc --nycrc-path=test/.nycrc-mocha mocha --colors",
```

Comment:
Why did you change normal tests to go through `nyc`? I tried `npm test` locally, and it replaces test output with coverage output (which is printed twice).

Reply:
I wanted coverage numbers to show up when running tests, it's a nice metric to have, and I removed all the noise that shows up with `npm run coverage` to keep it to the minimum when we run `npm test`.
Reason why it is printed twice is because client and server coverage are split, and `npm run coverage` is combining them for good measure.

I think it's pretty nice as is, and definitely worth it. I'd even prefer if it only printed one number in each coverage summary (so 1 number for client and 1 for server), but I don't know if `nyc`/Istanbul supports this.

Mind giving this a try, and we can revise/improve if this appears to not be optimal?